### PR TITLE
try to fix https://github.com/Microsoft/TypeScript/issues/8202

### PR DESCRIPTION
**Pull request by plantain-00**
Fixes https://github.com/Microsoft/TypeScript/issues/8202

### CODE REVIEW

File: baselines/dom.generated.d.ts [hunk 1]
```diff
@@ -9438,7 +9438,7 @@ declare var RTCRtpSender: {
 }
 
 interface RTCSrtpSdesTransport extends EventTarget {
-    onerror: ((ev: Event) => any) | null;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: we should not remove the ` | null` here.

File: baselines/dom.generated.d.ts [hunk 2]
```diff
@@ -9283,7 +9283,7 @@ declare var RTCDTMFToneChangeEvent: {
 
 interface RTCDtlsTransport extends RTCStatsProvider {
     ondtlsstatechange: ((ev: RTCDtlsTransportStateChangedEvent) => any) | null;
-    onerror: ((ev: Event) => any) | null;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: we should not remove the ` | null` here.

File: baselines/dom.generated.d.ts [hunk 3]
```diff
@@ -9338,7 +9338,7 @@ declare var RTCIceCandidatePairChangedEvent: {
 
 interface RTCIceGatherer extends RTCStatsProvider {
     readonly component: string;
-    onerror: ((ev: Event) => any) | null;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: and here

File: baselines/dom.generated.d.ts [hunk 4]
```diff
@@ -9397,7 +9397,7 @@ declare var RTCIceTransportStateChangedEvent: {
 }
 
 interface RTCRtpReceiver extends RTCStatsProvider {
-    onerror: ((ev: Event) => any) | null;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: and here.

File: baselines/dom.generated.d.ts [hunk 5]
```diff
@@ -9417,7 +9417,7 @@ declare var RTCRtpReceiver: {
 }
 
 interface RTCRtpSender extends RTCStatsProvider {
-    onerror: ((ev: Event) => any) | null;
```
Comment (mhegazy) [inline baselines/dom.generated.d.ts]: and here.

**Comment by mhegazy** [comment]
@zhengbli is there a better way to do this, may be on the generation script side?

**Comment by zhengbli** [comment]
This is easier to change from the script. I'll send up a PR shortly.

**Comment by mhegazy** [comment]
closing in favor of https://github.com/Microsoft/TSJS-lib-generator/pull/108

**Pull request closed without merging**